src/gleam/string.gleam

```diff
@@ -45,8 +45,15 @@ pub fn is_empty(str: String) -> Bool {
 ///    > length("")
 ///    0
 ///
-pub external fn length(String) -> Int =
-  "string" "length"
+if erlang {
+  pub external fn length(String) -> Int =
+    "string" "length"
+}
+
+if javascript {
+  pub external fn length(String) -> Int =
+  "" "String.prototype.length.call"
```

Comment:
I think this could fail if in the module there is a module imported under the name `String`. It would generate code like this:

```js
import * as String from "thestringmodule";

export function main() {
  return String.prototype.length.call("Hi");
}
```

Reply:
How would you access `String` if someones done that? `window.String`?

Reply:
Maybe? Not sure exactly what the JS scoping rules are. I would probably write a wrapper function